examples/simple-map-marker-static.html

```diff
@@ -11,6 +11,9 @@
   <link rel="import" href="../px-map-tile-layer.html" />
   <link rel="import" href="../px-map-marker-static.html" />
 
+  <link rel="import" href="../px-icon-set/px-icon-set.html"/>
```

Comment:
I don't think you need to import px-icon for all those demos since it is loaded by px-map